### PR TITLE
New section of Trento version history

### PR DESCRIPTION
**Pull request by tomschr**
### PR creator: Description

A section describing the different versions available of Trento along with their most important user-facing features.

### Confluence

* https://confluence.suse.com/display/TRNT/2023-08-16+Trento+Documentation

### CODE REVIEW

**Review by abravosuse — COMMENTED** [review]
@tomschr I have added initial content to this section. Please review when you get a chance. Thanks!

File: trento/xml/article_sap_trento.xml [hunk 1]
```diff
@@ -2517,21 +2517,101 @@ trento-server-rabbitmq-0</screen>
...
+        <term>Version 2.0.0</term>
+       <para> Consisting of Helm chart 2.0.0, web 2.0.0, cchecks engine (wanda) 1.0.0 and agent 2.0.0. Released on 2023/04/26.</para>
         <listitem>
-          <para></para>
+          <para>A brand-new safer, faster, ligher SSH-less configuration checks engine (wanda) which not only opens the door to configuration checks for other HA scenarios (ASCS/ER, HANA scaleup
+           cost optimized, HANA scale-out) and other targes in the environment (hosts, HANA databases, NetWeaver instances), but also will allow customization of existing checks and
+          addtion of custom checks.</para>
+        </listitem>
+               <listitem>
+          <para>Addtion of VMware to the list of known platforms.</para>
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:2571]: Typo:

```suggestion
          <para>Addition of VMware to the list of known platforms.</para>
```

File: trento/xml/article_sap_trento.xml [hunk 2]
```diff
@@ -2517,21 +2517,101 @@ trento-server-rabbitmq-0</screen>
...
+        </listitem>
+       <listitem>
+          <para>Versioned external APIs for both the web and the checks engine (wanda) components. The full list of available APIs can be found at 
+          https://www.trento-project.io/web/swaggerui/ and https://www.trento-project.io/wanda/swaggerui/ respectiverly.</para>
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:2575]: Use links:

```suggestion
          <link xlink:href="https://www.trento-project.io/web/swaggerui/"/> and <link xlink:href="https://www.trento-project.io/wanda/swaggerui/"/> respectiverly.</para>
```

File: trento/xml/article_sap_trento.xml [hunk 3]
```diff
@@ -2517,21 +2517,101 @@ trento-server-rabbitmq-0</screen>
...
+        </listitem>
+      </varlistentry>
+     <varlistentry>
+        <term>Version 1.2.0</term>
+       <para> Consisting of Helm chart 1.2.0, web 1.2.0, runner 1.1.0 and agent 1.1.0. Released on 2022/11/04.</para>
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:2580]: I think the version _and_ the release date are something the reader is interested in. As such, it should be more prominent and not hidden in a sentence:

```suggestion
       <term>Version 1.2.0, released on 2022/11/04</term>
       <para> Consisting of Helm chart 1.2.0, web 1.2.0, runner 1.1.0 and agent 1.1.0. </para>
```

File: trento/xml/article_sap_trento.xml [hunk 4]
```diff
@@ -2517,21 +2517,101 @@ trento-server-rabbitmq-0</screen>
...
+        <listitem>
+          <para>Configration checks for HANA scale-up peformance optimized two-node clusters on on-premise bare metal platforms, including KVM and Nutanix.</para>
+        </listitem>
+               <listitem>
+          <para>A dynamic dashboard that allows you to determine, at a glance, the overall health status of your SAP environment.</para>
+        </listitem>
+      </varlistentry>
+     <varlistentry>
+        <term>Version 1.1.0</term>
+       <para> Consisting of Helm chart 1.1.0, web 1.1.0, runner 1.0.1 and agent 1.1.0. Released on 2022/07/14.</para>
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:2590]:
```suggestion
        <term>Version 1.1.0, released on 2022/07/14</term>
       <para> Consisting of Helm chart 1.1.0, web 1.1.0, runner 1.0.1 and agent 1.1.0. </para>
```

File: trento/xml/article_sap_trento.xml [hunk 5]
```diff
@@ -2517,21 +2517,101 @@ trento-server-rabbitmq-0</screen>
...
+        <listitem>
+          <para>Fix for major bug in the runner component that prevented the web component to load properly.</para>
+        </listitem>
+      </varlistentry>
+     <varlistentry>
+        <term>Version 1.0.0 (general availability)</term>
+       <para> Consisting of Helm chart 1.0.0, web 1.0.0, runner 1.0.0 and agent 1.0.0. Released on 2022/04/29.</para>
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:2597]:
```suggestion
       <term>Version 1.0.0 (general availability), released on 2022/04/29</term>
       <para> Consisting of Helm chart 1.0.0, web 1.0.0, runner 1.0.0 and agent 1.0.0. </para>
```

File: trento/xml/article_sap_trento.xml [hunk 6]
```diff
@@ -2517,21 +2517,101 @@ trento-server-rabbitmq-0</screen>
...
+          <para>Host clean-up capabilities, allowing users to get rid of hosts that are no longer part of their SAP environment.</para>
+        </listitem>
+       <listitem>
+          <para>New checks results view that leverages the potential of the new checks engine (wanda) and provides the user with insightful information about the 
+           check execution.</para>
+        </listitem>
+      </varlistentry>
+           <varlistentry>
+        <term>Version 2.0.0</term>
+       <para> Consisting of Helm chart 2.0.0, web 2.0.0, cchecks engine (wanda) 1.0.0 and agent 2.0.0. Released on 2023/04/26.</para>
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:2564]:
```suggestion
        <term>Version 2.0.0, released on 2023/04/26</term>
       <para> Consisting of Helm chart 2.0.0, web 2.0.0, cchecks engine (wanda) 1.0.0 and agent 2.0.0.</para>
```

File: trento/xml/article_sap_trento.xml [hunk 7]
```diff
@@ -2517,21 +2517,101 @@ trento-server-rabbitmq-0</screen>
...
         <listitem>
-          <para></para>
+          <para>A brand-new safer, faster, ligher SSH-less configuration checks engine (wanda) which not only opens the door to configuration checks for other HA scenarios (ASCS/ER, HANA scaleup
+           cost optimized, HANA scale-out) and other targes in the environment (hosts, HANA databases, NetWeaver instances), but also will allow customization of existing checks and
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:2567]: Entities:

```suggestion
           cost optimized, &hana; scale-out) and other targes in the environment (hosts, &hana; databases, &netweaver; instances), but also will allow customization of existing checks and
```

File: trento/xml/article_sap_trento.xml [hunk 8]
```diff
@@ -2517,21 +2517,101 @@ trento-server-rabbitmq-0</screen>
...
+        <listitem>
+          <para>Fix for major bug in the runner component that prevented the web component to load properly.</para>
+        </listitem>
+      </varlistentry>
+     <varlistentry>
+        <term>Version 1.0.0 (general availability)</term>
+       <para> Consisting of Helm chart 1.0.0, web 1.0.0, runner 1.0.0 and agent 1.0.0. Released on 2022/04/29.</para>
+              <listitem>
+          <para>Clean, simple web console designed with SAP Basis in mind and that reacts, in real time, to changes happening in the 
+           backend environment thanks to the event-driven technology behind it.</para>
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:2600]: * Basis -> basis? If it's not a unique name, use lowercase.
* Split the long sentence in two.

```suggestion
          <para>Clean, simple web console designed with SAP basis in mind. It reacts in real time to changes happening in the 
           backend environment thanks to the event-driven technology behind it.</para>
```

File: trento/xml/article_sap_trento.xml [hunk 9]
```diff
@@ -2517,21 +2517,101 @@ trento-server-rabbitmq-0</screen>
...
+        </listitem>
+        <listitem>
+          <para>Automated discovery of SAP systems, SAP instances and SAP HANA scale-up clusters.</para>
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:2603]:
```suggestion
          <para>Automated discovery of SAP systems, SAP instances and &hana; scale-up clusters.</para>
```

File: trento/xml/article_sap_trento.xml [hunk 10]
```diff
@@ -2517,21 +2517,101 @@ trento-server-rabbitmq-0</screen>
...
+        </listitem>
+               <listitem>
+          <para>Configuration checks for HANA scale-up performance optimized two-node clusters on Azure, AWS and GCP.</para>
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:2606]:
```suggestion
          <para>Configuration checks for &hana; scale-up performance optimized two-node clusters on Azure, AWS and GCP.</para>
```

File: trento/xml/article_sap_trento.xml [hunk 11]
```diff
@@ -2517,21 +2517,101 @@ trento-server-rabbitmq-0</screen>
...
+        </listitem>
+               <listitem>
+          <para>Basic integration with Grafana and Prometheus to provide graphic dashboards about CPU and Memory utilization in each discovered host.</para>
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:2609]: Typo:

```suggestion
          <para>Basic integration with Grafana and Prometheus to provide graphic dashboards about CPU and memory utilization in each discovered host.</para>
```

**Comment by abravosuse** [comment]
Thanks for your suggestions @tomschr . I have reviewed and accepted them . I think it is ready to merge now.

File: trento/xml/article_sap_trento.xml [hunk 12]
```diff
@@ -2514,6 +2514,108 @@ trento-server-rabbitmq-0</screen>
...
+      The following list shows the most important user-facing features in the different versions
+     of Trento. For a more detailed information about the changes included in each new version, visit
+      the GitHub project at <link xlink:href="https://github.com/trento-project"/>. Particularly:
+    </para>
+   <itemizedlist>
+      <listitem>
+        <para>For changes in the Trento Helm Chart, visit <link xlink:href="https://github.com/trento-project/helm-charts/releases"/>.</para>
+      </listitem>
+      <listitem>
+        <para>For changes in the &t.server, web component, visit <link xlink:href="https://github.com/trento-project/web/releases"/>.</para>
```
Comment (abravosuse) [inline trento/xml/article_sap_trento.xml:2529]:
```suggestion
        <para>For changes in the &t.server; web component, visit <link xlink:href="https://github.com/trento-project/web/releases"/>.</para>
```

File: trento/xml/article_sap_trento.xml [hunk 13]
```diff
@@ -2514,6 +2514,108 @@ trento-server-rabbitmq-0</screen>
...
+          <listitem>
+        <para>For changes in the &t.agent;, visit <link xlink:href="https://github.com/trento-project/agent/releases"/>.</para>
+      </listitem>
+    </itemizedlist>
+    <!-- HINT: The list should be sorted by versions, having the most recent version at the
+               first item.
+    -->
+    <variablelist>
+      <varlistentry>
+        <term>Version 2.1.0</term>
```
Comment (abravosuse) [inline trento/xml/article_sap_trento.xml:2548]:
```suggestion
        <term>Version 2.1.0, released on 2023/08/02</term>
```

File: trento/xml/article_sap_trento.xml [hunk 14]
```diff
@@ -2514,6 +2514,108 @@ trento-server-rabbitmq-0</screen>
...
+       <para> Consisting of Helm chart 2.1.0, web 2.1.0, checks engine (wanda) 1.1.0 and agent 2.1.0. Released on 2023/08/02.</para>
```
Comment (abravosuse) [inline trento/xml/article_sap_trento.xml:2549]:
```suggestion
       <para> Consisting of Helm chart 2.1.0, web 2.1.0, checks engine (wanda) 1.1.0 and agent 2.1.0.</para>
```

**Review by abravosuse — CHANGES_REQUESTED** [review]

File: trento/xml/article_sap_trento.xml (hunk 12 above)
Comment (tomschr) [reply trento/xml/article_sap_trento.xml]: Is resolved now.

**Comment by tomschr** [comment]
@abravosuse I've fixed some validation issues in the list (it can be tricky). :wink: See e55f9c5.

I can provide you with a PDF if you like, otherwise I can merge it if you like.

**Comment by abravosuse** [comment]
@tomschr there is a change I made following your recommendation that is pending commit.

**Comment by tomschr** [comment]
> there is a change I made following your recommendation that is pending commit.

I've already taken into account.

Here is the result (HTML):

<details>
  <summary>Screenshot of HTML</summary>

Screenshot 2023-08-17 at 13-44-30 Getting started with Trento Premium


</details>

**Review by abravosuse — CHANGES_REQUESTED** [review]
@tomschr please review this

**Comment by tomschr** [comment]
@abravosuse Looks fine to me. :+1: Any objections to merge?

**Pull request merged**